application/org.openjdk.jmc.feature.flightrecorder/feature.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
    Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2022, Datadog, Inc. All rights reserved.
```

Comment:
Same here. Only add a Datadog (in this case) specific copyright line if it is a new file.